content/docs/SUSHI/project/_index.md

```diff
@@ -75,6 +61,10 @@ customized-ig
 └── sushi-config.yaml
 ```
 
+{{% alert title="Tip" color="success" %}}
+The IG templates define a set of supported input folders (e.g., input/profiles, input/resources, etc.) not shown above. If an author creates further subfolders (e.g., input/resources/nested), SUSHI will load these resources. However, since the IG publisher doesn't support those paths, it will emit a warning and exclude them from the IG JSON.
+{{% /alert %}}
```

Comment:
This is partly, but not entirely, true.  Authors can use the special IG parameter `path-resource` to tell the IG Publisher about other input paths to process, including nested paths. SUSHI is now smart enough to recognize that parameter and include resources in paths specified by that parameter (without emitting a warning)... 

But that's all a lot of detail to put in here.  Bottom line is that SUSHI processes paths as expected if the IG author knows how to use that `path-resource` parameter, and in other cases will warn the user if they use unsupported paths.

Reply:
Updated as follows:
```
The IG templates also define a set of supported input folders not shown above (e.g., input/profiles, input/resources, etc.). If there are additional subfolders (e.g., input/resources/nested), use the `path-resource` parameter in **sushi-config.yaml** to tell the IG Publisher which additional input paths to process (see [Specifying Additional Resource Paths](tips#specifying-additional-resource-paths) for details).
```